tools/lsp/ui/components/property-widgets.slint

```diff
@@ -668,6 +668,15 @@ export component PropertyValueWidget inherits HorizontalLayout {
     in property <bool> enabled;
     alignment: stretch;
 
+    function set-string-binding(text: string, is-translated: bool) {
```

Comment:
Unused argument looks suspicious. 
Why so much indirection? One need to look at many places to figure out what this does?  Can the caller not just call  set-code-binding?